Index: aula1-intro.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>a = 1\r\nb = 1.2\r\nc = 'G'\r\nghgh\r\nggfg\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/aula1-intro.py b/aula1-intro.py
--- a/aula1-intro.py	(revision eb4fdd88cbc1847d046d7ce8ddba1a7c530f0859)
+++ b/aula1-intro.py	(date 1611941581669)
@@ -1,5 +1,3 @@
 a = 1
 b = 1.2
 c = 'G'
-ghgh
-ggfg
